src/config.rs

```diff
@@ -340,4 +420,111 @@ mod tests {
         assert_eq!(args, FullCommandArgs::String("emacs -nw".to_owned()));
         assert_eq!(args.args(), ["emacs", "-nw"].as_ref());
     }
+
+    #[test]
+    fn test_layered_configs_resolved_config_values_empty() {
+        let empty_config = config::Config::default();
+        let layered_configs = LayeredConfigs {
+            default: empty_config.to_owned(),
+            env_base: empty_config.to_owned(),
+            user: None,
+            repo: None,
+            env_overrides: empty_config,
+            arg_overrides: None,
+        };
+        assert_eq!(layered_configs.resolved_config_values(&[]).unwrap(), []);
+    }
+
+    #[test]
+    fn test_layered_configs_resolved_config_values_single_key() {
+        let empty_config = config::Config::default();
+        let env_base_config = config::Config::builder()
+            .set_override("user.name", "base-user-name")
+            .unwrap()
+            .set_override("user.email", "base@user.email")
+            .unwrap()
+            .build()
+            .unwrap();
+        let repo_config = config::Config::builder()
+            .set_override("user.email", "repo@user.email")
+            .unwrap()
+            .build()
+            .unwrap();
+        let layered_configs = LayeredConfigs {
+            default: empty_config.to_owned(),
+            env_base: env_base_config,
+            user: None,
+            repo: Some(repo_config),
+            env_overrides: empty_config,
+            arg_overrides: None,
+        };
+        assert_eq!(
+            layered_configs.resolved_config_values(&[]).unwrap(),
+            [
+                // Note: "email" is alphabetized, before "name" from same layer.
```

Comment:
Nit: You might want to use `insta::assert_debug_snapshot!()`

Reply:
I did indeed! Thanks!